tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -99,3 +99,39 @@ describe('Test GitRepo Bundle name validation and max character trimming behavio
     }
   )
 });
+
+describe('Test resource behavior after deleting GitRepo using keepResources option', { tags: '@p1'}, () => {
+  const repoName = "local-cluster-fleet-71"
+  const branch = "master"
+  const path = "qa-test-apps/nginx-app"
+  const repoUrl = "https://github.com/rancher/fleet-test-data/"
+  const appNamespace = 'nginx-keep'
+  const appName = 'nginx-keep'
+  const keepResourceData: testData[] = [
+    { qase_id: 69,
+      keepResources: 'true',
+      test_explanation: 'RESOURCES will NOT be DELETED after GitRepo is deleted.',
+    },
+    { qase_id: 70,
+      keepResources: 'false',
+      test_explanation: 'RESOURCES will be DELETED after GitRepo is deleted.',
```

Comment:
Suggestion:
`RESOURCES will NOT BE KEPT and  WILL be DELETED after GitRepo is deleted`